qa/format.sh

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+# Utility file to run pre-commit hooks locally
+# Usage: bash qa/format.sh
+
+set -e
+
+pip install pre-commit
+pre-commit install
+pre-commit run --all-files
+rm .git/hooks/pre-commit*
```

Comment:
Do we need to install the pre-commit hooks if we're going to immediately remove them? I'm not sure about the proper approach, but the following works for me:
```suggestion
pip install pre-commit
pre-commit run --all-files
```
I don't see any changes in `.git/hooks`.